applications/nrf5340_audio/src/audio/le_audio_rx.c

```diff
@@ -139,24 +129,25 @@ void le_audio_rx_data_handler(uint8_t const *const p_data, size_t data_size, boo
 static void audio_datapath_thread(void *dummy1, void *dummy2, void *dummy3)
 {
 	int ret;
-	struct ble_iso_data *iso_received = NULL;
-	size_t iso_received_size;
+	struct audio_data *audio_frame = NULL;
+	size_t audio_frame_received_size;
```

Comment:
Picky: maybe we could rename "audio_frame_received_size" to "audio_frame_recv_size"